plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -164,3 +171,151 @@ store(
+			);
+			const { cartItem } = getContext< CartItemContext >();
+
+			return quantityDescriptionLabel.replace( '%s', cartItem.name );
+		},
+
+		// Intended to be used in context of a cart item in wp-each
+		get removeFromCartLabel() {
+			const { removeFromCartLabel } = getConfig(
+				'woocommerce/mini-cart-items-block'
+			);
+			const { cartItem } = getContext< CartItemContext >();
+
+			return removeFromCartLabel.replace( '%s', cartItem.name );
+		},
+
+		get cartItems() {
+			return wooStoreState.cart.items;
+		},
+
+		// Intended to be used in context of a cart item in wp-each
+		itemShortDescription() {
+			const ctx = getContext< CartItemContext >();
+			const el = getElement();
+
+			if ( el.ref ) {
+				const innerEl = el.ref.querySelector(
+					'.wc-block-components-product-metadata__description'
+				);
+
+				// A workaround for the lack of dangerous set HTML directive in interactivity API
+				if ( innerEl ) {
+					innerEl.innerHTML = ctx.cartItem.short_description;
+				}
+			}
+		},
+
+		// Intended to be used in context of a cart item in wp-each
+		get itemPrice(): string {
+			const ctx = getContext< CartItemContext >();
+			const { currency } = getConfig( 'woocommerce' );
+
+			const normalizedCurrency = normalizeCurrencyResponse(
+				wooStoreState.cart.totals,
+				currency
+			);
+
+			return formatPriceWithCurrency(
+				ctx.cartItem.prices.price,
+				normalizedCurrency
+			);
+		},
+
+		// Intended to be used in context of a cart item in wp-each
+		get lineItemTotal(): string {
+			const ctx = getContext< CartItemContext >();
+			const { displayCartPriceIncludingTax } = getConfig(
+				'woocommerce/mini-cart'
+			);
+			const { currency } = getConfig( 'woocommerce' );
+
+			const normalizedCurrency = normalizeCurrencyResponse(
+				wooStoreState.cart.totals,
+				currency
+			);
+
+			const totals = ctx.cartItem.totals;
+
+			const totalLinePrice = displayCartPriceIncludingTax
+				? parseInt( totals.line_subtotal, 10 ) +
+				  parseInt( totals.line_subtotal_tax, 10 )
+				: parseInt( totals.line_subtotal, 10 );
+
+			return formatPriceWithCurrency(
+				totalLinePrice,
+				normalizedCurrency
+			);
+		},
+
+		// Intended to be used in context of a cart item in wp-each
+		get itemThumbnail(): string {
+			const ctx = getContext< CartItemContext >();
+			return ctx.cartItem.images[ 0 ]?.thumbnail || '';
+		},
+	},
+	actions: {
+		*incrementQuantity(): Generator< unknown, void > {
+			const { cartItem } = getContext< CartItemContext >();
+			const { actions } = store< WooCommerce >(
+				'woocommerce',
+				{},
+				{ lock: universalLock }
+			);
+
+			yield actions.addCartItem( {
+				id: cartItem.id,
+				quantity: cartItem.quantity + 1,
+			} );
+		},
+
+		*decrementQuantity(): Generator< unknown, void > {
+			const { cartItem } = getContext< CartItemContext >();
+			const { actions } = store< WooCommerce >(
+				'woocommerce',
+				{},
+				{ lock: universalLock }
+			);
+
+			yield actions.addCartItem( {
```

Comment:
I realize this API as is, is not well suited to this purpose. I'll do some additions to the store itself in a separate follow up PR.